dowhy/causal_graph.py

```diff
@@ -84,26 +84,16 @@ def view_graph(self, layout="dot"):
             self.logger.warning("Warning: Pygraphviz cannot be loaded. Check that graphviz and pygraphviz are installed.")
             self.logger.info("Using Matplotlib for plotting")
             import matplotlib.pyplot as plt
-            solid_edges = [(n1,n2) for n1,n2, e in self._graph.edges(data=True) if 'style' not in e ]
-            dashed_edges =[(n1,n2) for n1,n2, e in self._graph.edges(data=True) if ('style' in e and e['style']=="dashed") ]
-            plt.clf()
```

Comment:
are these 2 lines on clf and shell_layout no longer needed?

Reply:
I've added the changes, after adjusting for the syntax